build.gradle

```diff
@@ -72,6 +72,7 @@ project.ext {
 
 configurations {
     provided
+    packagingOnly
```

Comment:
This is not used anywhere

Reply:
i guess provided is neither...

Reply:
yes, you just have to rebase, then provided is removed